lms/djangoapps/certificates/api.py

```diff
@@ -259,3 +274,27 @@ def get_active_web_certificate(course, is_preview_mode=None):
         if config.get('is_active') or is_preview_mode:
             return config
     return None
+
+
+def emit_certificate_event(event_name, user, course_id, course=None, event_data=None):
+    """
+    Emits certificate event.
+    """
+    event_name = '.'.join(['edx', 'certificate', event_name])
+    if course is None:
+        course = modulestore().get_course(course_id, depth=0)
+    context = {
+        'user_id': user.id,
+        'org_id': course.org,
+        'course_id': unicode(course_id)
```

Comment:
@ziafazal -- after taking a closer look, we do not need to include the 'path' parameter in this event payload